app/views/learning/_card.html.slim

```diff
@@ -5,6 +5,6 @@
       div class='govuk-card__hero govuk-!-margin-bottom-2'
         img.full-width-img src=image_path(mod.thumbnail) width=200 alt=mod.name
       .govuk-card__content
-        h3.govuk-heading-m= mod.title
+        h3.govuk-heading-m= "Module #{mod.id}: #{mod.title}"
```

Comment:
I use something very similiar on the learning log that I moved to the training module class `tab_label`; perhaps repurpose that.  see https://github.com/DFE-Digital/early-years-foundation-recovery/blob/main/app/views/notes/show.html.slim

Reply:
I'd normally favour keeping things that are just view layer decoration separate from a db model object. We are at the cusp of this it seems with our `TrainingModule` and though it isn't an `ActiveRecord` model we are treating it as such. Would anyone object to the idea?

Reply:
What idea?  I'd prefer this to be in the model, as that will be the first place I would look and not scattered across multiple views.

Reply:
If we are to start to have multiple methods that concatenate strings, for example, just for the view, rather than have them be on the model, move them to a decorator that wraps it.

Reply:
I don't like moving to a decorator before it's really warranted - belongs on the model until the need arises.

Reply:
I'm going to resolve this so I can merge